docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -31,6 +31,30 @@ command:
 
     openssl rand 512 | base64
 
+.. _verify_secrets:
+
+Verify configuration
+^^^^^^^^^^^^^^^^^^^^
+
+To verify shared secret configuration:
+
+1. Start your Trino cluster with two or more nodes configured with a shared
+   secret.
+2. Connect to the :doc:`Web UI </admin/web-interface>` from your browser using a
```

Comment:
Remove "using a URL such as ..."

I am not even sure we need the rest of the sentences .. the Web UI docs should already explain how to log in so we dont need to repeat it here